packages/telemetry/test/browser.spec.ts

```diff
@@ -0,0 +1,137 @@
+// import { expect } from 'chai'
```

Comment:
Uncomment file?

Reply:
+ 1 I'd like to see these tests run if possible

Reply:
yeah, this is a todo... i got sidetracked but these tests will be completed before this is merged.

Reply:
I to did it!